Library/Homebrew/formula_installer.rb

```diff
@@ -1024,13 +1025,20 @@ def link(keg)
   end
 
   sig { void }
-  def install_plist
-    return unless formula.plist
+  def install_service
+    plist = if formula.plist
```

Comment:
I wonder if `service` should take priority over `plist` (as it's newer). Similarly, we should probably error out on the formula DSL if both are specified.